JS/navigation.js

```diff
@@ -0,0 +1,10 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+    function openNav() {
```

Comment:
Brug en eventlistener i stedet for functions
```suggestion
   let nav = document.querySelector(".nav");
   nav.addEventListener("click", (event) => {
       if(event.target.className == "nav__open") {
           nav.style.width = "250px";
       }
       else if(event.target.className == "nav__close") {
           nav.style.width = "0";
       }
   }
```